extensions/guacamole-auth-radius/src/main/java/org/apache/guacamole/auth/radius/AuthenticationProviderService.java

```diff
@@ -53,6 +54,12 @@
      * Logger for this class.
      */
     private final Logger logger = LoggerFactory.getLogger(AuthenticationProviderService.class);
+    
+    /**
+     * The name of the password field for the user to response to the RADIUS
```

Comment:
> The name of the password field for the user to response to the RADIUS challenge.

I was initially going to say that "response" should be "respond" here, but with that change the sentence still reads a bit off ("the password field for the user to respond").

Did you mean the password field that will contain the user's response?

Reply:
Fixed this up.